contracts/JBFundingCycleStore.sol

```diff
@@ -256,8 +249,8 @@ contract JBFundingCycleStore is JBControllerUtility, IJBFundingCycleStore {
     // Duration must fit in a uint64, and must be greater than 1000 seconds to prevent manipulative miner behavior.
     require(_data.duration <= type(uint64).max && _data.duration > 1000, '0x15: BAD_DURATION');
 
-    // Discount rate token must be less than or equal to 100%. A value of 1000000001 means non-recurring.
-    require(_data.discountRate <= 1000000001, '0x16: BAD_DISCOUNT_RATE');
+    // Discount rate token must be less than or equal to 100%.
+    require(_data.discountRate <= 100000000, '0x16: BAD_DISCOUNT_RATE');
```

Comment:
Consider extracting this into a meaningful constant, MAX_DISCOUNT_RATE, for better readability. I see it's used in many places throughout file, so we should replace them there with the new constant.

Reply:
done